### DISCUSSION TITLE
Sphinx Docstring parameter descriptions "key" and "keyword" do not put a new line in "rendered" docstring.

### DISCUSSION THREAD

**Discussion by jeremyvital4** [microsoft/pylance-release#2689] [Enhancement]
<h3> Environment data </h3>

-   Language Server version: 2021.10.4-pre.1
-   OS and version: darwin x64
-   Python version (and distribution if applicable, e.g. Anaconda): 
-   python.analysis.indexing: undefined
-   python.analysis.typeCheckingMode: basic

<h3> Expected behaviour </h3>

`:key kwarg1:` should appear on a new line, as it is valid format for ReST:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#info-field-lists
`param, parameter, arg, argument, key, keyword: Description of a parameter.`

<h3> Actual behaviour </h3>

keyword args denoted with `:key kwarg1:` is not put on a new line
![image](https://user-images.githubusercontent.com/88157114/139321173-a4cbbb3c-da2b-4960-9a15-b2e2c9ac0c72.png)

<h3> Logs </h3>

<h3> Code Snippet / Additional information </h3>

```
def test_function(arg1, arg2, arg3, **kwargs):
    """
    description

    :param arg1: first arg
    :type arg1: int
    :param arg2: second arg
    :type arg2 int
    :param arg3: third arg
    :type arg3: str
    :key kwarg1: a keyword arg
    :type kwarg1: bool
    :key kwarg2: another keyword arg
    :type kwarg2: bool
    """
    pass
```

![image](https://user-images.githubusercontent.com/88157114/139321163-45ba4bf3-5815-4892-9716-effb68528f0a.png)

**Comment by yvvt0379** [comment]
I'm also having the similar issue , see #1662

**Comment by RyskDigital** [comment]
As a Sphinx user it would definitely be very helpful to have some of the basics of the reST format be supported for docstring rendering on hover.

**Comment by cullenwren95** [comment]
Is there presently an extension that creates the desired docstring display behavior?

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
